# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/nsINetworkInterceptController.idl
# Commit: 55f13fb4ee3f
# Full Hash: 55f13fb4ee3f06209d00fd91a7a03884bab88e2d
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-12-15 09:27:59
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Implement csp for Module Workers; r=evilpie,asuth,rpl,ckerschb
#   
#   Depends on D155691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156102
# ==============================================================================

diff -r 354711cf113a -r 55f13fb4ee3f netwerk/base/nsINetworkInterceptController.idl
--- a/netwerk/base/nsINetworkInterceptController.idl	Wed Dec 14 14:55:37 2022 +0000
+++ b/netwerk/base/nsINetworkInterceptController.idl	Wed Dec 14 14:55:38 2022 +0000
@@ -171,7 +171,8 @@
           case nsIContentPolicy::TYPE_INTERNAL_SCRIPT_PRELOAD:
           case nsIContentPolicy::TYPE_INTERNAL_MODULE:
           case nsIContentPolicy::TYPE_INTERNAL_MODULE_PRELOAD:
-          case nsIContentPolicy::TYPE_INTERNAL_WORKER_IMPORT_SCRIPTS: {
+          case nsIContentPolicy::TYPE_INTERNAL_WORKER_IMPORT_SCRIPTS:
+          case nsIContentPolicy::TYPE_INTERNAL_WORKER_STATIC_MODULE: {
             aKey = "subresource-script"_ns;
             break;
           }